gokart/parameter.py

```diff
@@ -5,11 +5,22 @@
 import luigi
 from luigi import task_register
 
+import gokart
+
 logger = getLogger(__name__)
 
 
 class TaskInstanceParameter(luigi.Parameter):
 
+    def __init__(self, expected_type=None, *args, **kwargs):
+        if expected_type is None:
+            self.expected_type = gokart.TaskOnKart
+        elif isinstance(expected_type, type):
+            self.expected_type = expected_type
+        elif expected_type is not None:
```

Comment:
```suggestion
        else:
```

If there're no need for special care, just putting else can work.